src/common/run_command.h

```diff
@@ -5,10 +5,60 @@
 #ifndef BITCOIN_COMMON_RUN_COMMAND_H
 #define BITCOIN_COMMON_RUN_COMMAND_H
 
+#if defined(HAVE_CONFIG_H)
+#include <config/bitcoin-config.h>
+#endif
+
+#include <util/fs.h>
+#include <util/strencodings.h>
+
 #include <string>
 
+#if defined(ENABLE_EXTERNAL_SIGNER) && defined(BOOST_POSIX_API)
+#include <fcntl.h>
+#ifdef FD_CLOEXEC
+#include <unistd.h>
+#if defined(__GNUC__)
```

Comment:
In-code warning suppression has [recently been removed](https://github.com/bitcoin/bitcoin/pull/28002), so you can drop all of the `#if defined(__GNUC__)` blocks.